src/tests/modify.test.js

```diff
@@ -396,4 +396,44 @@ describe('modify() - basic mutations', () => {
       },
     });
   });
+
+  it('customize x-jsf-presentation (shorthand)', () => {
+    const result = modify(schemaPet, {
+      fields: {
+        pet_age: () => {
+          return {
+            presentation: {
+              'data-foo': 123,
+            },
+          };
+        },
+        'pet_address.street': {
+          errorMessage: {
+            'data-foo': 456,
+          },
+        },
+      },
+    });
+
+    expect(result.schema).toMatchObject({
```

Comment:
Note: I've rebased with `main` https://github.com/remoteoss/json-schema-form/pull/83 and had to update this test to match the new specs. I need a re-approval